headers/types/common/common.h

```diff
@@ -136,14 +136,21 @@ struct rect16_xywh {
 };
 ASSERT_SIZE(struct rect16_xywh, 8);
 
-// The fact the 4th byte is never used hints to the fact the struct is packed
-struct rgb {
+struct rgb_24 {
```

Comment:
How about we call this `rgb`, and the other one `rgbx`? It's clearer, and the `_<number>` suffix is a convention I'd like to reserve for the enum wrappers that #22 introduces.

Reply:
Sure thingy ^^ I mostly avoided using that name because  there were problems with it at first